packages/shared/src/components/cards/v1/PostCardFooter.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { CardImage, CardVideoImage } from './Card';
+import { Post, isVideoPost } from '../../../graphql/posts';
+import { cloudinary } from '../../../lib/image';
+
+interface PostCardFooterClassName {
+  image?: string;
+}
+
+type PostCardFooterProps = {
+  insaneMode: boolean;
+  openNewTab: boolean;
+  showImage: boolean;
+  post: Post;
+  className: PostCardFooterClassName;
+};
+
+export const PostCardFooter = ({
+  post,
+  showImage,
+  className,
+}: PostCardFooterProps): ReactElement => {
+  const isVideoType = isVideoPost(post);
+  const ImageComponent = isVideoType ? CardVideoImage : CardImage;
+
+  if (!showImage) {
+    return null;
+  }
+  return (
+    <>
+      {/* {!showImage && post.author && (
+        <PostAuthor
+          author={post.author}
+          className={classNames(
+            'mx-4 mt-2 hidden tablet:flex laptop:hidden',
+            visibleOnGroupHover,
+          )}
+        />
+      )} */}
```

Comment:
Can be removed? Or should be brought back?

Reply:
Same as above

Reply:
If we want to leave these commented I think we better add a TODO for the person working on the task after to check, as just commented it might go unnoticed. Or we bring it back and leave it for the right card to owner to decide what needs to be done.